wikicharts/content_gap_plots.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        #chart.plot_monthlyscatter('time', 'metric_underrepresented', wmf_colors['blue'])
</pre><p>Remove commented out code unless there's a good reason to keep it (which should be explained in a commented).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>